### PR TITLE
[FLINK-15190][doc]add documentation for DDL in FLIP-69

### PR DESCRIPTION
**Pull request by zjuwangg**
## What is the purpose of the change

*add documentation for DDL in FLIP-69*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? ( docs )

### CODE REVIEW

**Comment by zjuwangg** [comment]
cc @bowenli86 @xuefuz @lirui-apache to have a review.

File: docs/dev/table/sql.md [hunk 1]
```diff
@@ -1324,7 +1324,7 @@ MATCH_RECOGNIZE (
 
 ## DDL
 
-DDLs are specified with the `sqlUpdate()` method of the `TableEnvironment`. The method returns nothing for a success table creation. A `Table` can be register into the [Catalog](catalogs.html) with a `CREATE TABLE` statement, then can be referenced in SQL queries in method `sqlQuery()` of `TableEnvironment`.
+DDLs are specified with the `sqlUpdate()` method of the `TableEnvironment`. The method returns nothing for a success create/drop/alter database or table. A `Table` can be register into the [Catalog](catalogs.html) with a `CREATE TABLE` statement, then can be referenced in SQL queries in method `sqlQuery()` of `TableEnvironment`.
```
Comment (bowenli86) [inline docs/dev/table/sql.md]:
```suggestion
DDLs are specified with the `sqlUpdate()` method of the `TableEnvironment`. The method returns nothing for a success create/drop/alter database or table operation. A `CatalogTable` can be register into the [Catalog](catalogs.html) with a `CREATE TABLE` statement, then can be referenced in SQL queries in method `sqlQuery()` of `TableEnvironment`.
```

File: docs/dev/table/sql.md [hunk 2]
```diff
@@ -1434,6 +1434,79 @@ If the table does not exist, nothing happens.
...
+
+* Rename Table
+
+{% highlight sql %}
+ALTER TABLE [catalog_name.][db_name.]table_name RENAME TO new_table_name
+{% endhighlight %}
+
+Rename the given table name to another new table name.
+
+* Set Table Properties
```
Comment (bowenli86) [inline docs/dev/table/sql.md]:
```suggestion
* Set or Alter Table Properties
```

File: docs/dev/table/sql.md [hunk 3]
```diff
@@ -1434,6 +1434,79 @@ If the table does not exist, nothing happens.
 
 {% top %}
 
+### Alter Table
+
+* Rename Table
+
+{% highlight sql %}
+ALTER TABLE [catalog_name.][db_name.]table_name RENAME TO new_table_name
```
Comment (bowenli86) [inline docs/dev/table/sql.md]: do we support [IGNORE IF NOT EXISTS] here?

File: docs/dev/table/sql.md [hunk 4]
```diff
@@ -1434,6 +1434,79 @@ If the table does not exist, nothing happens.
...
+**CASCADE**
+
+Dropping a non-empty database also drops all associated tables and functions.
+
+{% top %}
+
+### Alter Database
+
+{% highlight sql %}
+ALTER DATABASE [catalog_name.]db_name SET (key1=val1, key2=val2, ...)
```
Comment (bowenli86) [inline docs/dev/table/sql.md]: do we support [IGNORE IF NOT EXISTS] here?

File: docs/dev/table/sql.zh.md [hunk 5]
```diff
@@ -29,7 +29,7 @@ This is a complete list of Data Definition Language (DDL) and Data Manipulation
...
 
-In order to access a table in a SQL query, it must be [registered in the TableEnvironment](common.html#register-tables-in-the-catalog). A table can be registered from a [TableSource](common.html#register-a-tablesource), [Table](common.html#register-a-table), [CREATE TABLE statement](#create-table), [DataStream, or DataSet](common.html#register-a-datastream-or-dataset-as-table). Alternatively, users can also [register catalogs in a TableEnvironment](catalog.html) to specify the location of the data sources.
+In order to access a table in a SQL query, it must be [registered in the TableEnvironment](common.html#register-tables-in-the-catalog). A table can be registered from a [TableSource](common.html#register-a-tablesource), [Table](common.html#register-a-table), [CREATE TABLE statement](#create-table), [DataStream, or DataSet](common.html#register-a-datastream-or-dataset-as-table). Alternatively, users can also [register catalogs in a TableEnvironment](catalogs.html) to specify the location of the data sources.
```
Comment (bowenli86) [inline docs/dev/table/sql.zh.md]: I think what's missing is the difference between a temp table and a catalog table. Maybe we should have a dedicated page `Tables` for such concept explanation, similar to `Functions` page

File: docs/dev/table/sql.zh.md [hunk 6]
```diff
@@ -1324,7 +1324,7 @@ MATCH_RECOGNIZE (
 
 ## DDL
 
-DDLs are specified with the `sqlUpdate()` method of the `TableEnvironment`. The method returns nothing for a success table creation. A `Table` can be register into the [Catalog](catalogs.html) with a `CREATE TABLE` statement, then can be referenced in SQL queries in method `sqlQuery()` of `TableEnvironment`.
+DDLs are specified with the `sqlUpdate()` method of the `TableEnvironment`. The method returns nothing for a success create/drop/alter database or table. A `Table` can be register into the [Catalog](catalogs.html) with a `CREATE TABLE` statement, then can be referenced in SQL queries in method `sqlQuery()` of `TableEnvironment`.
```
Comment (bowenli86) [inline docs/dev/table/sql.zh.md]:
```suggestion
DDLs are specified with the `sqlUpdate()` method of the `TableEnvironment`. The method returns nothing for a success create/drop/alter database or table operation. A `CatalogTable` can be register into the [Catalog](catalogs.html) with a `CREATE TABLE` statement, then can be referenced in SQL queries in method `sqlQuery()` of `TableEnvironment`.
```

File: docs/dev/table/sql.md (hunk 3 above)
Comment (zjuwangg) [reply docs/dev/table/sql.md]: We don't support  [IGNORE IF NOT EXISTS] here

File: docs/dev/table/sql.md (hunk 4 above)
Comment (zjuwangg) [reply docs/dev/table/sql.md]: We don't support  [IGNORE IF NOT EXISTS] here

File: docs/dev/table/sql.zh.md (hunk 5 above)
Comment (zjuwangg) [reply docs/dev/table/sql.zh.md]: Right. We need a dedicated page to introduce such  concepts.

**Comment by bowenli86** [comment]
LGTM, merging

please consider FLINK-15192 next

**Pull request closed without merging**